steps/synopsys-ptpx-genlibdb/configure.yml

```diff
@@ -49,7 +49,6 @@ preconditions:
   - assert Tool( 'pt_shell' )                # tool check
   - assert File( 'inputs/adk' )              # must exist
   - assert File( 'inputs/design.vcs.v' )     # must exist
-  - assert File( 'inputs/design.spef.gz' )   # must exist
```

Comment:
Can you actually leave this in (but commented out), and then replace "must exist" with "spef is optional"?

That way people don't think we just forgot and then decide to add it back in.

Reply:
Done. Actually DC is generating a spef file in the report.
I added some step to compress this file in .gz form to be used